wrappers/rust/README.md

```diff
@@ -0,0 +1,32 @@
+# indy
```

Comment:
* Add notes about experimental status of rust wrapper here
* look at original readme of rust wrapper to find more details and include it here